sub-logs/cliqq.json

```diff
@@ -0,0 +1,15 @@
+{
+  "owner": {
+    "username": "JemPH",
+    "email": "jamusicfinder@gmail.com"
+  },
+
+  "target": {
+    "record-type": {
```

Comment:
You must provide the record type in this entry, e.g. CNAME, A, AAAA etc. Please correct this.